.aspect/workflows/bazelrc

```diff
@@ -1,6 +1,6 @@
 # build without the bytes
-common --remote_download_minimal
-common --nobuild_runfile_links
+test --remote_download_minimal
```

Comment:
What was the reason for this change?

Reply:
Looks like a bug in Bazel for the handling of `common`. It turns out that `bazel query` doesn't like `common --remote_download_minimal`. We now do a `bazel query` in the delivery manifest step.